### PR TITLE
Ana/3313 add link for linux users to ledger connection issues article

### LINKED ISSUE #3313: add message for Linux users with a link to ledger connection issues article
**Is your feature request related to a problem? Please describe.**

since we switched to WebUSB, linux users will experience a connection issue with their devices. 

**Describe the solution you'd like**

let's add a message for linux users informing them of this change with a link to ledger connection issue article with steps for setting up ledger in the CLI.

**Describe alternatives you've considered**

we've considered switching all linux users to web HID but since some users may have gone through the connection issue article already, we think we should wait to switch to HID when chrome adds this to the browser natively, or maybe when the cosmos ledger app 2 is released and we'll have to communicate accordingly anyways.

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #3313 

**Description:**
Added the message for Linux users with the link to Ledger documentation.

Still, I am having an issue with the new test. It says:
```
[Vue warn]: Failed to resolve directive: clipboard
```
which doesn't make much sense since it is exactly the same logic being used in the Windows bit and there it doesn't error :woman_shrugging: 
The test itself does pass, it is just this weird directive thing failing.

(I also centered the SignIn button since I find it more beautiful so)

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by jbibla** [comment]
this is excellent!

File: tests/unit/specs/components/common/TmSessionHardware.spec.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ import TmSessionHardware from "common/TmSessionHardware"
 const localVue = createLocalVue()
 localVue.use(Vuex)
 localVue.use(Vuelidate)
-localVue.directive(`tooltip`, () => { })
-localVue.directive(`focus`, () => { })
+localVue.directive(`tooltip`, () => {})
```
Comment (jbibla) [inline tests/unit/specs/components/common/TmSessionHardware.spec.js]: the issue with the clipboard can be solved by adding this: `localVue.directive(`clipboard`, () => {})`

Comment (jbibla) [reply tests/unit/specs/components/common/TmSessionHardware.spec.js]: or this: `localVue.directive('clipboard', VueClipboard)`

Comment (Bitcoinera) [reply tests/unit/specs/components/common/TmSessionHardware.spec.js]: Thank you Jordan! I just fixed it :v:

**Pull request merged**